tests/TestCase/Console/ShellTest.php

```diff
@@ -824,6 +825,28 @@ public function testRunCommandHittingTaskInSubcommand()
     }
 
     /**
+     * test calling a shell that dispatch another one
+     *
+     * @return void
+     */
+    public function testDispatchShell()
+    {
+        $Shell = new TestingDispatchShell();
+        ob_start();
+        $Shell->runCommand(['test_task'], true);
+        $result = ob_get_clean();
+
+        $expected = <<<TEXT
+<info>Welcome to CakePHP v3.0.1 Console</info>
+I am a test task, I dispatch another Shell
+<info>Welcome to CakePHP v3.0.1 Console</info>
```

Comment:
This test will fail every time we make a new release. Could we do a series of `assertContains()` instead so we can avoid brittleness with the version number.